examples/bridging/withdraw_and_claim.py

```
@@ -0,0 +1,204 @@
+            # Find our withdrawal (most recent matching amount)
+            target_tx = None
+            for tx in history:
+                # Basic matching logic: Same amount, status logic
+                if tx['amount'] == amount_str and tx['type'] == 'withdrawal':
+                    target_tx = tx
+                    break
+            
+            if target_tx:
+                status = target_tx['status']
+                if status == 'completed': # Ready to claim
+                    print(f"\n[+] Withdrawal ready! Fetching proof...")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)